src/components/PeopleList.tsx

```diff
@@ -0,0 +1,88 @@
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  listOfPeople: Person[];
+};
+
+export const PeopleList: React.FC<Props> = ({ listOfPeople }) => {
+  const { selectedSlug = 0 } = useParams();
+
+  const list = listOfPeople.map(personage => {
```

Comment:
use camel case for variables 

```suggestion
  const list = listOfPeople.map(personAge => {
```